engine/Default/shop_goods.php

```diff
@@ -35,7 +35,8 @@
 	if ($port->hasGood(GOODS_NARCOTICS))
 		$base_chance -= 4;
 
-	if ($ship->isUnderground()) {
+	//IF evil ship OR blockade runner THEN reduce chance of being caught with illegal goods
+	if ($ship->isUnderground() || ($ship->getShipTypeID() == 66)) {
```

Comment:
No need to change it while we're in the draft phase of this PR, but for future reference we want to avoid numeric references to, well, pretty much everything! In this case, the `SHIP_TYPE_BLOCKADE_RUNNER` constant should be used instead.